src/units/attack_type.hpp

```diff
@@ -354,6 +364,7 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	std::string icon_;
 	std::string range_;
 	int min_range_, max_range_;
+	std::string attack_alignment_;
```

Comment:
IMO this should also be just `alignment_`.

Reply:
if alignment_ variable exist in unit class i prefer let the actual name.

Reply:
i prefer be careful and use a diffrent name what for type and unit class

Reply:
Why is that helpful? An attack also has `id_`, `type_`, `description_`, etc all without an attack_ prefix even though unit has those as well. To me it seems more confusing to special case alignment.

Reply:
changed to alignment_